lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java

```diff
@@ -39,8 +39,7 @@
 import org.hibernate.Query;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
-import org.hibernate.criterion.DetachedCriteria;
-import org.hibernate.criterion.Restrictions;
+import org.hibernate.criterion.*;
```

Comment:
We generally prefer not using .* especially in lessons since so many class names overlap each other and have to be fully expanded anyway. I think it should be converted back since you only appear to be using the 2 you removed anyway.